adding_species_manuscript.tex

```diff
@@ -667,6 +665,7 @@ \subsection*{Filling out the missing pieces}
 So, the benefit of including species with very partial genome builds in \stdpopsim
 would be outweighed by the substantial extra burden on \stdpopsim maintainers as well as
 downstream users of these models.
+% Izabel: this bit above is a repetition, maybe delete or shrink? Or maybe when talking about it in the chromosome part just briefly mention it. 
```

Comment:
I like having a concluding sentence here. I think it does a decent job in summarizing the various points raised in this paragraph. maybe it could be shortened a bit...